src/ibm/hypervisor-network-mgr-src/xyz.openbmc_project.Network.Hypervisor.service.in

```diff
@@ -1,6 +1,9 @@
 [Unit]
 Description=Hypervisor Network Manager
+Wants=xyz.openbmc_project.biosconfig_manager.service
```

Comment:
The changes here will most likely cause this service, xyz.openbmc_project.Network.Hypervisor.service, to start later in the boot to BMC Ready. As long as there were no expectations that it start sooner in the boot, these changes are fine. If however something like bmcweb was assuming the d-bus interfaces provided by this service were available early on, it could cause new issues in that area.